src/get.js

```diff
@@ -0,0 +1,29 @@
+const https = require('https');
+
+const baseUrl = 'https://rickandmortyapi.com/api/';
+
+function get(endpointUrl) {
+  return new Promise(function (resolve, reject) {
+    const url = `${baseUrl}${endpointUrl}`;
+    https.get(url, function (response) {
+      response.setEncoding('utf8');
+      let body = '';
+
+      response.on('data', (data) => (body += data));
+
+      response.on('error', (error) => reject(error));
+
+      response.on('end', function () {
+        body = JSON.parse(body);
+        response.data = body;
+        if (response.statusCode >= 200 && response.statusCode < 300) {
+          resolve(response);
+        } else {
+          reject(response);
+        }
```

Comment:
Please note: I tried to mimic the behavior your existing tests were expecting. Feel free to offer any suggestions or improvements.

Reply:
Thank you for that, looks good.